javascript2-week1/script.js

```diff
@@ -0,0 +1,20 @@
+document.getElementById('randomize').addEventListener('click', function() {
+    let answers = document.querySelectorAll('.answers div');
+    for (let i = answers.length - 1; i > 0; i--) {
+        let j = Math.floor(Math.random() * (i + 1));
+        [answers[i].innerHTML, answers[j].innerHTML] = [answers[j].innerHTML, answers[i].innerHTML];
```

Comment:
Something here does not work correctly, when I press the randomize button all the values from the inputs disapear. This code is also very heard to read and hard to understand what it does. I think the goal of the task was to keep the input values and have a random order of those.